README.md

```diff
@@ -69,10 +74,10 @@ $ cd UploadToCounty
 $ python UploadToPostgresCountiesZips.py
 ```
 
-## Install JS dependencies
+## Install JS dependencies (make sure npm is installed)
 ```sh
 $ cd frontend
-$ yarn install
+$ npm install
```

Comment:
Let's make sure we keep `yarn` instead of `npm`.  The dependency tree resolution is better and it forces a change in practice in committing the `.lock` file.

Reply:
fixed